src/vision.cs

```diff
@@ -4457,6 +4502,10 @@ interface VNHumanBodyPose3DObservation {
 		[return: NullAllowed]
 		[return: BindAs (typeof (VNHumanBodyPose3DObservationJointName))]
 		NSString GetParentJointName ([BindAs (typeof (VNHumanBodyPose3DObservationJointName))] NSString jointName);
+
+		[Export ("getCameraRelativePosition:forJointName:error:")]
+		// MarshalDirective not needed bc Matrix4 param is passed by reference as opposed to by value as typically done
+		bool GetCameraRelativePosition (out Matrix4 modelPositionOut, [BindAs (typeof (VNHumanBodyPose3DObservationJointName))] NSString jointName, [NullAllowed] out NSError error);
```

Comment:
Since you already have a test app, would it be possible to write a manual test for this (I'm assuming you'll need a photo for this; that can be added to the monotouch-test project as a BundleResource - the only caveat would be to try to shrink the image if possible to avoid multi-mb files in the repo :))

Reply:
@rolfbjarne yep I can do that. with the api doing significantly better when the image has depth data and also having to comply with license/copyright restrictions, I need to use an image of our own..I will try to recruit a team member from the office next week for the sample image 😀